### PR TITLE
feat: add role name to json object

### LINKED ISSUE #6014: Add role name in role-invite
Add role name attribute in the json response on acceptance of a role.

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #6014

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Adds a role-name attribute to the json object. As suggested here fossasia/open-event-frontend#2771

### CODE REVIEW

File: app/api/role_invites.py [hunk 1]
```diff
@@ -168,7 +168,8 @@ def accept_invite():
     return jsonify({
         "email": user.email,
         "event": role_invite.event_id,
-        "name": user.fullname if user.fullname else None
+        "name": user.fullname if user.fullname else None,
+        "role": uer.role.name
```
Comment (prateekj117) [inline app/api/role_invites.py]: Fix the typo.

Comment (uds5501) [reply app/api/role_invites.py]: @prateekj117 It's not a typo :laughing: , it's a `uer` object, `users_events_roles`

**Comment by uds5501** [comment]
@mrsaicharan1 @shreyanshdwivedi @prateekj117 please review!

File: app/api/role_invites.py (hunk 1 above)
Comment (kushthedude) [reply app/api/role_invites.py]: @uds5501 Can you name more understandable,At first even i thought its a typo

Comment (uds5501) [reply app/api/role_invites.py]: @kushthedude I think that wouldn't exactly be necessary. all through our code, `uer` is being used as `users_events_roles` model so I think I should stick to it to maintain uniformity.

**Pull request merged**